### DISCUSSION TITLE
Application colors

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#75] [Q&A] [answered]
Would you like more color in the user interface?
Is the appearance more attractive and can the categories be seen better?
(similar is the dark app theme)

<img src="https://user-images.githubusercontent.com/3484020/225854856-de63a396-fa89-48d8-977c-5afa7231f815.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/224285279-a8f8324a-0db8-4814-aba0-e90951537ee5.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/224285271-40b12d70-073f-4742-9071-e05b904e2de1.jpg" width="33%"/>

**Comment by airald** [comment]
My oppinion is that colors are useful to identify functions. Blue is good as generic app colour, but, some screens can have other to increase useability.

**Comment by devemux86** [comment]
It is the "accent" color, used everywhere in the app.

If you find it difficult to read, we can make the color darker or lighter?
Unfortunately it is not easy to let users change it.

Likewise for the dark app theme, its color is slightly different.

**Comment by Leeb64** [comment]
There's nothing wrong with the different colors in the user interface. But for me, different colored asterisks for different user favorite categories would be more helpful.
This annoying swapping of the user POI would then no longer be necessary...

**Reply by devemux86** [reply]
> But for me, different colored asterisks for different user favorite categories would be more helpful.
This annoying swapping of the user POI would then no longer be necessary...

Favorites are already being redesigned, also because of the new Search in https://github.com/devemux86/cruiser/discussions/74.
You can expect many new things in the future.

We can better discuss the favorites in https://github.com/devemux86/cruiser/discussions/45. :slightly_smiling_face:

**Comment by DolomitesTom** [comment]
looks good to me

**Comment by MotoUKRider** [comment]
My phone's Status Bar (when viewing the map) remains black whilst in portrait, but turns "Cruiser blue" when in landscape.  Is this deliberate?

**Reply by devemux86** [reply]
The dark app theme has black status bar, the light app theme has blue status bar, in both orientations.

What Android and device?
Can you post a screenshot?

**Reply by MotoUKRider** [reply]
Yes, see below - ignore the left-hand black edge of the landscape image as that's part of the screenshot image (I'm guessing because the camera eye is part of the screen).

This is a Samsung S20, Android v13 / UI v5.  I've noticed this for a while, but can't really call it an issue, more "odd"!

Map view, a few seconds after running Cruiser:
![cruiser-statusbar-1](https://user-images.githubusercontent.com/125696109/226132772-0de647ba-dcf0-4d1c-a5a9-12d52ec3e625.jpg)

Rotate the phone to landscape, blue!
![cruiser-statusbar-2](https://user-images.githubusercontent.com/125696109/226132774-1d8ffcc2-ed09-4df7-9a63-e74989715fdd.jpg)

**Reply by devemux86** [reply]
This is strange, I need to search it.

If you open the settings in portrait mode, what color do they have?

**Reply by 0709wiwiwi** [reply]
By phone Xiamo Mi A2 Lite > A10.
- Color change (Black/Blue) with app theme light.
- Color change (Black/Blue) with app theme system.
- No color change (Black/Black ) with app theme black.

No issue for me either.
In settings app theme light and system:  White background and blue top border. (no change when rotating)
In setting app theme black: Black background and black top border. (no change when rotating)

**Reply by MotoUKRider** [reply]
Blue top in the settings screen :)

**Reply by devemux86** [reply]
@MotoUKRider 

Have you changed anything in the Android display settings or the Samsung has any special theme settings?

Have you changed anything in the Android developer settings?
(if you enabled and know them)

**Reply by MotoUKRider** [reply]
Interesting. It's not you, it's me!  On my Galaxy Tab, it's fine - blue on both orientations.  I *do* have Developer Settings enabled, mostly to install my own APKs, but noting else as far as I can tell.  Stock ROM and updated when I'm nagged etc.

**Reply by 0709wiwiwi** [reply]
**Phone** Xiamo Mi A2 Lite. (A10)
Developer options: ON > Select app for nep location: Lockito.
Cruiser check: Colour change as reported before

Check by Developer options: OFF.
After restart phone:
Cruiser check: Color change as reported before

**Phone** Motorola G42 (A12)
Developer options: ON > Select app for nep location: Lockito.
Exact same phenomen.
Cruiser check: Color change as reported before

**Tablet** Samsung TAB4 (7 ")  SM-T235 (A5.1.1)
Developper options: ON
Cruiser check: No color change.

**Tablet** Samsung TAB Pro (10.1 ")  SM-T520 (A7.1)
Developper options: ON
Cruiser check: No color change.

**Comment by devemux86** [comment, marked as answer]
Available in [Cruiser 3.0.13](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by devemux86** [comment]
Do you use the "Settings | Application | Appearance | Menu color" to change the color of the menu screens?
Or you keep the default blue / cyan app color for the menu buttons?

I am wondering if this color setting is really needed?

**Reply by WolfgangZ42** [reply]
Don't use it and think it's not necessary. Just my 2 cents.

**Reply by GuG1050** [reply]
For me also not needed.

**Reply by DolomitesTom** [reply]
I would like to keep it, even if I use it not too often. Sometimes I like to change colours. It doesn't disturb at all, why delete it?
I only find that the settings symbols colour should be also changed by this menu colour button. At the moment this remains in blue

**Reply by devemux86** [reply]
> I only find that the settings symbols colour should be also changed by this menu colour button. At the moment this remains in blue

This is the accent color of the app.
Android does not seem to allow changing it, unless I find a "smart" way to do it. :slightly_smiling_face: 

That's why I wonder if the menu color setting is really needed.
We better use the accent color and find a "way" to change it.

**Comment by Ciutnik** [comment]
Whenever possible, I try to set the maximum contrast between the text and the background. I admit I initially changed the color of the icons from cyan to white (I have a dark theme permanently set), but the discrepancy with the accent color annoyed me so I went back to the default cyan :-)
 
So in the future (this may well be in the distant future!) I would only change the color if there was consistency between the accent color and the icon color. But if there was any problem with it, it would probably be a waste of trouble and unnecessary complication (e.g. changing this color when changing between a light and dark theme of the application).

**Reply by devemux86** [reply]
> but the discrepancy with the accent color annoyed me

:+1: 
This was the reason of my [question](https://github.com/devemux86/cruiser/discussions/75#discussioncomment-6023280)!

**Reply by Ciutnik** [reply]
> We better use the accent color and find a "way" to change it.

Your answer is probably the best answer to your question 🙂
